mesonbuild/modules/external_project.py

```diff
@@ -93,7 +93,7 @@ def __init__(self,
         # will install files into "c:/bar/c:/foo" which is an invalid path.
         # Work around that issue by removing the drive from prefix.
         if self.prefix.drive:
-            self.prefix = self.prefix.relative_to(self.prefix.drive)
+            self.prefix = Path('/' + str(self.prefix.relative_to(self.prefix.drive + "/")))
```

Comment:
This mixes the new addition of single-quoted strings and double-quoted strings on the same line! :) Please be consistent (meson mostly consistently uses single-quotes).

Reply:
Done at 9f2acb2